app/Actions/AfterCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Actions;
+
+use App\Support\BaseAction;
+
+class AfterCommands extends BaseAction
+{
+    /**
+     * Perform the After Commands in Project Directory.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $directory  = config('lambo-store.project_path');
+
+        $commands   = config('lambo-after.commands');
```

Comment:
Please no matched indent

Reply:
Yes, I tend to do that.. Also have to stop doing it. Sure.

Reply:
Looks like also changes not pushed here